ctlcmd/cmdbase.py

```diff
@@ -1021,88 +1023,63 @@ def parse(self, args):
     if args.x or args.y:
       raise ValueError('You can either specify det-id or x y, not both')
 
-    if not args.detid in self.board.dets():
+    if not args.detid in self.board.get_all_detectors():
```

Comment:
`get_all_detectors` Now returns a list of detectors while `args.detid` is an `int`, so we probably want something like `if not args.detid < len(self.board.get_all_detectors())`